api/ipfsproxy/ipfsproxy.go

```diff
@@ -515,6 +519,40 @@ func (proxy *Server) repoStatHandler(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
+// repoGcHandler launches the garbage collection sequence in the cluster.
+// Based on https://docs.ipfs.io/reference/api/http/#api-v0-repo-gc
+func (proxy *Server) repoGcHandler(w http.ResponseWriter, r *http.Request) {
+	//TODO: Need help/direction on finishing the handler
+	peers := make([]peer.ID, 0)
+	err := proxy.rpcClient.CallContext(
+		proxy.ctx,
+		"",
+		"Cluster",
+		"IPFSRepoGc",
+		&peers,
```

Comment:
I just used this from repo stat, not sure if this is the right way to do it. Can somebody confirm if this is alright?